platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -2607,6 +2610,10 @@ static int tm_capabilities(odp_tm_capabilities_t capabilities[],
 		per_level_cap->max_priority       = ODP_TM_MAX_PRIORITIES - 1;
 		per_level_cap->min_weight         = ODP_TM_MIN_SCHED_WEIGHT;
 		per_level_cap->max_weight         = ODP_TM_MAX_SCHED_WEIGHT;
+		per_level_cap->min_burst          = 0;
+		per_level_cap->max_burst          = UINT32_MAX;
+		per_level_cap->min_rate           = TM_MIN_SHAPER_BW * 8U * 1000U;
+		per_level_cap->max_rate           = TM_MAX_SHAPER_BW * 8U * 1000U;
```

Comment:
This overflows in a system with 32 bit long ints. ULL suffix should make the literals large enough.